docs/posts/llms-and-data/index.qmd

```diff
@@ -0,0 +1,258 @@
+---
+title: LLMs and data
+author: "Cody Peterson"
+date: "2023-09-27"
+freeze: auto
+categories:
+    - blog
+    - ai
+    - llm
+---
+
+## Introduction
+
+While large-language models (LLMs) have been around for years, recent versions
+and innovations have made it possible to create instruction-following,
+conversational bots that can perform tasks on behalf of the user. The
+thought of using natural language to transform and analyze data is appealing.
+
+## Approaches
+
+When discussed internally at Voltron Data, we identified three distinct
+approaches to applying LLMs to data analytics that can be implemented today:
+
+1. LLM writes SQL
```

Comment:
```suggestion
1. LLM writes an analytic routine (SQL query or dataframe code)
```

Reply:
Probably don't need the parenthetical. How about just `LLM writes a SQL query or dataframe code`?

Reply:
yep lemme rework per this suggestion